packages/flutter/lib/src/widgets/committed_content.dart

```diff
@@ -0,0 +1,54 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart';
+
+/// A class representing data for the `onContentCommitted` callback on text fields.
+///
+/// This will be used when content is inserted into a text field. The class holds
+/// information for the mime type, URI (location), and bytedata for the inserted
+/// content.
+@immutable
+class CommittedContent {
```

Comment:
Is the named after the Android method `InputConnection.commitContent`? I'd recommend something like `InputContent` or `CommittedContentFromInputMethod`, `CommittetContent` doesn't feel that self-explanatory.

Reply:
I renamed pretty much all the methods and classes to `onContentInserted`, `contentInsertionMimeTypes`, and `KeyboardInsertedContent` - I think those should be more self-explanatory now.